Gemfile

```diff
@@ -1,30 +1,30 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
```

Comment:
- [ ] Please make sure that the app works properly. Currently, there's a "cors" error (see attached below). You can know more about "cors" and how to fix it from [here](https://www.stackhawk.com/blog/rails-cors-guide/).

Reply:
resolved! ✔️